src/all/akuma/src/eu/kanade/tachiyomi/extension/all/akuma/Akuma.kt

```diff
@@ -208,8 +300,85 @@ class Akuma : ParsedHttpSource() {
         return document.select(".entry-content img").attr("abs:src")
     }
 
+    override fun getFilterList(): FilterList = FilterList(
+        Filter.Header("Separate tags with commas (,)"),
+        Filter.Header("Prepend with dash (-) to exclude"),
+        TextFilter("Female Tags", "female"),
+        TextFilter("Male Tags", "male"),
+        CategoryFilter(),
+        TextFilter("Groups", "group"),
+        TextFilter("Artists", "artist"),
+        TextFilter("Parody", "parody"),
+        TextFilter("Characters", "character"),
+        Filter.Header("Filter by pages, for example: (>20)"),
+        TextFilter("Pages", "pages"),
+        Filter.Header("Search in favorites, read, or commented"),
+        OptionFilter(),
+
+        Filter.Separator(),
+        Filter.Header("Use syntax to search. If not blank, other filters will be ignored"),
+        SyntaxFilter(),
```

Comment:
what is syntax filter for? if you mean raw search syntax from the site then it can be used in regular query.

Reply:
I forgot. sorry.